server/svix-server/src/core/security.rs

```diff
@@ -60,6 +61,9 @@ impl Permissions {
 pub struct CustomClaim {
     #[serde(rename = "org", default, skip_serializing_if = "Option::is_none")]
     pub organization: Option<String>,
+
+    #[serde(rename = "evt_ff", default, skip_serializing_if = "Option::is_none")]
+    pub event_feature_flags: Option<EventTypeFeatureFlagSet>,
```

Comment:
As said above, it's not only for events. It's a general feature flag mechanism. Name it accordingly.

Reply:
Same as above, renamed to `feature_flags`